code/utils/include/allscale/utils/vector.h

```diff
@@ -104,6 +108,14 @@ namespace utils {
 			return out << vec.data;
 		}
 
+#if defined(ALLSCALE_WITH_HPX)
```

Comment:
In theory it would work. I decided not to do that here. The reason is, `Vector` is not a `std::array`, by explicitly dispatching it, we don't loose anything, but automatically gain the potential bitwise copy serialization. The problem is/was, that the allscale specialization is disabled for `Vector`/`std::array` when serialization is turned on. I would rather keep it that way.